src/gitops/hooks/useApplicationActionsProvider.tsx

```
@@ -151,20 +151,21 @@ export const useApplicationActionsProvider: UseApplicationActionsProvider = (app
-        label: t('View in Argo CD'),
-        accessReview: {
-          group: ApplicationModel.apiGroup,
-          verb: 'get' as K8sVerb,
-          resource: ApplicationModel.plural,
-          namespace: application?.metadata?.namespace,
-        },
-        cta: () => {
-          window.open(href, '_blank');
-        },
-      },
+      ...(href ? [{
```

ESLint

[error] Replace `·?·[` with `⏎········?·[⏎············`

(prettier/prettier)